_includes/components/header.html

```diff
@@ -0,0 +1,162 @@
+{% if header %}
+  <div class="usa-overlay"></div>
+{% if header.type == 'basic' %}
+  <header class="usa-header usa-header--basic" role="banner">
+{% elsif header.type == 'basic-mega' %}
+  <header class="usa-header usa-header--basic usa-header--basic-megamenu" role="banner">
+{% elsif header.type == 'extended' or header.type == 'extended-mega' %}
+  <header class="usa-header usa-header--extended" role="banner">
+{% endif %}
+
+  {% if header.type == 'basic' or header.type == 'basic-mega' %}
+    <div class="usa-nav-container">
+  {% endif %}
+    <div class="usa-navbar">
+      <div class="usa-logo" id="header-logo">
+        <a
+          href="{% if header.href %}{{ header.href }}{% else %}{{ site.baseurl }}/{% endif %}"
+          title="Home">
+          {% if header.logo %}
+            <img
+              class="usa-logo-img"
+              src="{% if header.logo.external %}{{ header.logo.src }}{% else %}{{ header.logo.src | relative_url }}{% endif %}"
+              alt="{{ header.logo.alt }}">
+          {% endif %}
+          <!-- <em class="usa-logo__text">
```

Comment:
Let's remove this and comment that this is the difference between the `uswds-jekyll` template and the one here.